alpa/global_env.py

```diff
@@ -20,10 +20,21 @@ def __init__(self):
         self.shard_parallel_sync_for_timer = False
 
         ########## Options of pipeline_parallel ##########
-        self.profile_with_whole_ray_cluster = True
+        # Whether to debug with local runtime. The local runtime checks
+        # correctness of stage construction and other graph level operations.
         self.debug_with_local_runtime = False
```

Comment:
Is `debug_with_local_runtime` used? What's the difference between `debug_with_local_runtime` and `debug_with_pipeshard_runtime`

Reply:
With `debug_with_pipeshard_runtime`, we can dump everything(`ReshardingTask`, `PipelineInst`, `ExecutableConfig` etc.) without physical resources. `LocalRuntime` aims to check the correctness of passes on jaxprs.

Reply:
I think `debug_with_local_runtime` is never used in the current codebase

Reply:
I'll deprecate it later

Reply:
Is the local runtime still there? I think we can deprecate all those code